pkg/apiserver/api_installer.go

```diff
@@ -321,10 +321,12 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 
 			// list or post across namespace.
 			// TODO: more strongly type whether a resource allows these actions on "all namespaces" (bulk delete)
-			namer = scopeNaming{scope, a.group.Linker, gpath.Join(a.prefix, itemPath), true}
-			actions = appendIf(actions, action{"LIST", resource, params, namer}, isLister)
-			actions = appendIf(actions, action{"POST", resource, params, namer}, isCreater)
-			actions = appendIf(actions, action{"WATCHLIST", "watch/" + resource, params, namer}, allowWatchList)
+			if !hasSubresource {
+				namer = scopeNaming{scope, a.group.Linker, gpath.Join(a.prefix, itemPath), true}
+				actions = appendIf(actions, action{"LIST", resource, params, namer}, isLister)
+				actions = appendIf(actions, action{"POST", resource, params, namer}, isCreater)
```

Comment:
Sadly, it looks like you're right:
https://github.com/GoogleCloudPlatform/kubernetes/blob/5520386b180d3ddc4fa7b7dfe6f52642cc0c25f3/plugin/pkg/scheduler/factory/factory.go#L311
https://github.com/GoogleCloudPlatform/kubernetes/blob/52db576617a5d0c10d9ea30ab031f8f4609ddaf4/contrib/mesos/pkg/scheduler/plugin.go#L119
https://github.com/GoogleCloudPlatform/kubernetes/blob/241e87cf9b4bfd810fcd92141ee70ff7a201a96b/contrib/mesos/pkg/executor/executor.go#L470

The client method exists but appears to not be used:
https://github.com/GoogleCloudPlatform/kubernetes/blob/5520386b180d3ddc4fa7b7dfe6f52642cc0c25f3/pkg/client/pods.go#L112